src/main/java/org/jabref/gui/importer/actions/MergeReviewIntoCommentAction.java

```diff
@@ -20,7 +20,7 @@ public void performAction(BasePanel basePanel, ParserResult parserResult) {
 
         migration.performMigration(parserResult);
         List<BibEntry> conflicts = MergeReviewIntoCommentMigration.collectConflicts(parserResult);
-        if (new MergeReviewIntoCommentConfirmationDialog(basePanel).askUserForMerge(conflicts)) {
+        if (!conflicts.isEmpty() && new MergeReviewIntoCommentConfirmationDialog(basePanel).askUserForMerge(conflicts)) {
```

Comment:
I think, this makes it not better. Now, JabRef migrates WITHOUT ANY notification. The old way was better. It showed that something was migrated.

Reply:
Well, then lets discuss this again: The rationale behind #3658 was that the warning should be displayed iff there is BOTH a comment and and review field present and perform the migration silently nonetheless.